algopy_testing/src/algopy/__init__.py

```diff
@@ -0,0 +1,14 @@
+# ruff: noqa
+
+# TODO: uncomment as new features are implemented
```

Comment:
Rather than a bunch of TODOs I was thinking of having a script that checks the stubs against the testing implementations and outlines any stubs that aren't covered.